chromium/storage.js

```diff
@@ -1,6 +1,10 @@
 var storage = chrome.storage.local;
```

Comment:
This entire file can be deleted and replaced with simple line in each file `storage` is used.

`const storage = chrome.storage.sync || chrome.storage.local`